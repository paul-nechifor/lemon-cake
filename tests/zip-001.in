(zip :(1 2) :(a b))
(zip :(1 2) :(a b) :(A B))
(zip :(1 2 c) :(a b) :(A B))
(zip :(1 2 c) :(a b c) :(A B))
(zip :(1 2 c) :(a b c) :(A B c))
