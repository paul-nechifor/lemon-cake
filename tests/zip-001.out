((1 a) (2 b))
((1 a A) (2 b B))
((1 a A) (2 b B))
((1 a A) (2 b B))
((1 a A) (2 b B) (c c c))
